distributions/standalone/src/main/resources/apps/IGNORE

```diff
@@ -0,0 +1,3 @@
+Git does not allow versioning of an empty folder so we add this file here to create the folder in git.
```

Comment:
This file should not go into the distribution. If there is not way to remove the file and keep the folder in the distro, the rename the file to readme.txt of something like that